types/eth1.proto

```diff
@@ -63,7 +69,7 @@ message Eth1Transaction {
     uint64 commulative_gas_used = 18;
     uint64 gas_used = 19;
     bytes logs_bloom = 20;
-    uint64 status = 21;
+    StatusType status = 21;
```

Comment:
We need to make sure that this doesn't cause any conflict when using proto buffer

Reply:
I suggest adding this as new field instead of replacing the existing one, otherwise this might cause issues with data that is currently in the db?